discovery-provider/alembic/versions/561ba1eca13a_add_slot_index.py

```diff
@@ -0,0 +1,23 @@
+"""Add Slot Index
+
+Revision ID: 561ba1eca13a
+Revises: 7693ab16e2e1
+Create Date: 2021-09-20 21:06:07.682836
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "561ba1eca13a"
+down_revision = "7693ab16e2e1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_index(op.f("ix_plays_slot"), "plays", ["slot"], unique=False)
```

Comment:
should this be 'if exists' or smth? so it doesn't fail on the existing nodes

Reply:
not sure what you mean - wouldn't the migrations run sequentially, ensuring that the column _does_ in fact exist?

Reply:
yeah but we manually added this index on foundation nodes, so it might fail if we do it again no? like 'create_index' that already exists - unless that doesn't fail

Reply:
Ohhh gotcha gotcha. I named this one ix_plays_slot which i think is different, i'll test to see if it collides

Reply:
will still add IF NOT EXISTS

Reply:
several ways of doing this. what you're planning is fine @rickyrombo 👍